lib/tasks/client_generate.rake

```diff
@@ -61,15 +61,15 @@ class ClientGenerator
 
     msg("\nGenerating API Ruby Client ...")
     generate_yaml_file(openapi_file, openapi_yaml_file)
-    system("java -jar #{generator_cli_jar} generate -i #{openapi_yaml_file} -c #{generator_config} -g ruby -o #{client_dir}")
+    system("java -jar #{generator_cli_jar} generate -i #{openapi_yaml_file} -c #{generator_config} -g #{lng} -o #{client_dir}")
   end
 end
 
 namespace :client do
-  desc "Generate the Toplogical Inventory API Ruby Client"
-  task :generate, [:client_dir] => [:environment] do |_task, args|
+  desc "Generate the Topological Inventory API Ruby Client (default is Ruby)"
+  task :generate, [:client_dir, :language] => [:environment] do |_task, args|
     default_client_dir = Pathname.new(Rails.root.join("..", "topological_inventory-api-client-ruby"))
```

Comment:
hmm, this should probably change  to -api-client-#{language}

Reply:
done